rust/Dockerfile

```diff
@@ -1,9 +1,10 @@
-ARG ALPINE_VERSION=3.19.2
-ARG CARGO_CHEF_VERSION="0.1.62"
+ARG RUST_VERSION=1.79.0
+ARG ALPINE_VERSION=3.20
```

Comment:
Patch versions [don't seem to be tagged](https://hub.docker.com/_/rust/tags?page=&page_size=&ordering=&name=alpine) in the `rust` repo...